_boolean.py

```diff
@@ -0,0 +1,19 @@
+from cellprofiler_core.setting.text import Integer
+
+
+class Boolean(Integer):
```

Comment:
You didn't want to use `BinaryChoice`? 😆

I would happily accept this into `cellprofiler_core` if you're interested in pushing it upstream.

Reply:
@0x00b1 Happily! I wasn't sure if I was going overboard with the settings subclsasses. PR filed: https://github.com/CellProfiler/core/pull/76